src/platform/intel/ace/platform.c

```diff
@@ -52,6 +54,16 @@ static const struct sof_ipc_fw_ready ready
 	.flags = DEBUG_SET_FW_READY_FLAGS,
 };
 
+void platform_panic(uint32_t p)
+{
+	mailbox_sw_reg_write(SRAM_REG_FW_STATUS, p & 0x3fffffff);
+	struct ipc_cmd_hdr header;
+	header.pri = 0x80000000 | (p & 0x3fffffff);
+	header.ext = MAILBOX_EXCEPTION_OFFSET + 2 * 0x20000;
```

Comment:
No magic values please

Reply:
Same is done for apollolake, cannonlake, icelake... Just following the convention :-)